azurerm/resource_arm_kubernetes_cluster_test.go

```diff
@@ -526,9 +523,6 @@ resource "azurerm_kubernetes_cluster" "test" {
 	
   network_profile {
     network_plugin     = "kubenet"
-    dns_service_ip     = "10.10.0.10"
-    docker_bridge_cidr = "172.18.0.1/16"
-    service_cidr       = "10.10.0.0/16"
   }
```

Comment:
we should update to make two tests here, one for when this is not specified and one for when this is

Reply:
Yeah, 2 legacy tests will use different settings: one is all empty, the other is all non-empty.

---
In reply to: [207462036](https://github.com/terraform-providers/terraform-provider-azurerm/pull/1715#discussion_r207462036) [](ancestors = 207462036)